Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -78,6 +78,11 @@ public function index(Document $document, $indexName)
                 case Field::TYPE_ARRAY:
                     $fields[$massiveField->getName()] = $value;
                     break;
+                case Field::TYPE_DATE:
```

Comment:
exactly for that i have introduced the converter mechanism why don`t you use it?